.changelog/v1.0.0/improvements/ibc-relayer/2754-Split-gm-out-of-the-hermes-repo

```diff
@@ -0,0 +1,4 @@
+- Removed `gm` folder and it's contents
```

Comment:
This should probably go in `.changelog/v1.1.0`

Reply:
[66f18a7](https://github.com/informalsystems/hermes/pull/2775/commits/66f18a7fa5b60958e450288b63f2cd9856490513)